chrome/extension/inject.js

```diff
@@ -4,18 +4,16 @@ function updateStyles() {
     (request, sender) => {
       if (!sender.tab) {
         if (request.update === 'style') {
-          const property = request.property;
-          const el = document.querySelectorAll(request.selector);
-          const color = request.color;
+          const { property, selector, color } = request;
+          const el = document.querySelectorAll(selector);
           let rgba;
 
           if (color) {
             rgba = `rgba(${color.r},${color.g},${color.b},${color.a})`;
           }
 
-          el.forEach(target => {
-            const element = target;
-            if (request.color) {
```

Comment:
Well actually just looked at it, and think this was the only place it was relevant, so when we've made sure runHelpers is all good I'll merge this